packages/devtools_app/web/debugger_screen.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+
+<!--
+  Copyright 2018 The Chromium Authors. All rights reserved.
+  Use of this source code is governed by a BSD-style license that can be
+  found in the LICENSE file.
+-->
+<html>
+
+<head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script>
+        const GA_DEVTOOLS_PROPERTY = 'UA-26406144-34'; // Dart DevTools GA Property UA.
+
+        function getDevToolsPropertyID() {
+            return GA_DEVTOOLS_PROPERTY;
+        }
+    </script>
+    <!-- The below URI ?id= must match the GA_DEVTOOLS_PROPERTY above. -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-26406144-34"></script>
+    <script type="text/javascript" src="devtools_analytics.js"></script>
+    <!-- End of DevTools Google Analytics -->
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="favicon.png" rel="icon" sizes="64x64">
+    <link href="packages/octicons_css/octicons_4.4.0.css" rel="stylesheet">
+    <link href="packages/polymer_css/flex.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/debugger/debugger.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/inspector/inspector.css" rel="stylesheet">
```

Comment:
remove the inspector, logging, performance, profiler, info, and timeline
css files.

Reply:
Done.